charts/spire/charts/spire-server/templates/issuer.yaml

```diff
@@ -0,0 +1,13 @@
+{{- with .Values.upstreamAuthority.certManager }}
```

Comment:
This is a great idea.  I would like something a bit more unified though, such that turning on this cert manager guarantees that any other approaches are deactivated.

Reply:
If you turn on multiple upstreamAuthorities, the helm chart fails to install. So I think you're covered already. I had disk and certManager activated, and it was failing to start.

https://github.com/spiffe/helm-charts/blob/main/charts/spire/charts/spire-server/templates/configmap.yaml#L103

Reply:
@edwbuck we could add the check for multiple upstreams as part of this pr: https://github.com/spiffe/helm-charts/pull/229 or as a followon. It lays some framework to make it easier.

Reply:
It already has a check for multiple upstreams. I would not duplicate that work. Link above

Reply:
Heh. I totally forgot I wrote that.... Never mind. :)